packages/forms/src/Components/ColorPicker.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Filament\Forms\Components;
+
+use Closure;
+
+class ColorPicker extends Field
+{
+    use Concerns\HasAffixes;
+    use Concerns\HasExtraAlpineAttributes;
+    use Concerns\HasExtraInputAttributes;
+    use Concerns\HasPlaceholder;
+    use Concerns\CanBeInline;
+
+    protected string $view = 'forms::components.color-picker';
+
+    protected string | Closure $format = 'hex';
+
+    protected bool | Closure $preview = true;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->inline(false);
+    }
+
+    public function format(string | Closure $format): static
```

Comment:
Could we get some extra shortcut methods for doing `->format('hex')`, etc?

Reply:
Sure!

Reply:
Done, added 
`->hex()`
`->hsl()`
`->rgb()`
`->rgba()`